src/DataAccess/Extensions/ApprenticeshipExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using SFA.DAS.Apprenticeships.DataAccess.Entities.Apprenticeship;
+
+namespace SFA.DAS.Apprenticeships.DataAccess.Extensions
+{
+    internal static class ApprenticeshipExtensions
+    {
+        //todo given this is now looking at future and past episodes to find the relevant one to authorise against, do we want to rename this method? GetRelevantEpisodeForAuthorisation is a bit of a mouthful but more descriptive
+        //todo check this can run inside a Linq to entities predicate
```

Comment:
It can't, but I think that is ok as we need a few values from the episode in memory anyway and we are only pulling a single episode from the DB.